docs/Documentation/UserGuideV0.7.0/2-Concept Key Concepts and Terminology/3-Encoding.md

```diff
@@ -7,9 +7,9 @@
     to you under the Apache License, Version 2.0 (the
     "License"); you may not use this file except in compliance
     with the License.  You may obtain a copy of the License at
-
+    
```

Comment:
Apache format is specified, why adds the extra `tab`?

Reply:
Done. Thx.